doc/development/developer_guide.md

```diff
@@ -0,0 +1,187 @@
+# Developer Guide
+
+This guide provides development workflows that are used to develop and maintain
+the import-export-udf-common-scala. It is intended for those who wish to address
+the issues, merge a pull request, perform release or deep dive into the codebase
+of the project.
+
+## Contributing
+
+Contributions to the import-export-udf-common-scala projects are very welcome!
+
+Please feel free to report a bug, suggest an idea for a feature, or ask a
+question about the code.
+
+Please keep in mind that contributions are not only pull requests. They can be
+any helpful comment on issues, improving documentation, enhancing the build process
+and many other tasks.
+
+### Getting in touch
```

Comment:
I think we usually use a title case in markdown headers. Do you want to change them?

Reply:
Done